src/main/java/frc/robot/Robot.java

```diff
@@ -190,10 +200,23 @@ public CommandBase releaseBalls(int ballCount) {
         return cmd;
     }
 
+    public CommandBase systemsCheck() {
+        CommandBase cmd = new SequentialCommandGroup(intake.intake(), indexer.pierrePossesionNEW(),
```

Comment:
Don't name something "new", it's not very descriptive. Either get rid of the old one and use that name (if it's a non breaking change) or come up with a more descriptive name for it